exercises/custom-set/example.py

```diff
@@ -0,0 +1,42 @@
+class CustomSet(object):
+    def __init__(self, elements=[]):
+        self.elements = list(elements)
+
+    def empty(self):
+        return not any(self.elements)
+
+    def __iter__(self):
+        return iter(self.elements)
```

Comment:
`__iter__` is a kind of useful method to implement here, but I think that `__contains__` would be a more logical for testing whether a set contains a particular value. Presumably `__iter__` is providing that functionality here, but from a learner point of view, I don't think that this would be clear enough.

Reply:
I somehow was not aware of the `__contains__` method previously, and though that implementing `__iter__` was the correct way of supporting the `in` operator... I will fix this, thanks!